samples/client/petstore/swift5/urlsessionLibrary/.openapi-generator/FILES

```diff
@@ -2,64 +2,64 @@
 Cartfile
 Package.swift
 PetstoreClient.podspec
```

Comment:
@aymanbagabas I almost forgot that we need to update the `podspec` to match the new file structure.
Could you please update the property `source_files` in the `podspec` file?
Thanks